codes.json

```diff
@@ -732,6 +732,14 @@
       "en": "MapMonitoring status must be either enabled or disabled (case insensitive)."
     }
   },
+  "InvalidFormat": {
```

Comment:
this is returned if the user tries to export to xls for example, or any other format other than csv or xlsx